packages/@aws-cdk/aws-redshift-alpha/lib/cluster.ts

```diff
@@ -558,6 +565,17 @@ export class Cluster extends ClusterBase {
       );
     }
 
+    const nodeType = props.nodeType || NodeType.DC2_LARGE;
+
+    if (props.multiAz) {
+      if (!nodeType.startsWith('ra3')) {
+        throw new Error(`Multi-AZ cluster is only supported for RA3 node types. got: ${nodeType}`);
+      }
+      if (clusterType === ClusterType.SINGLE_NODE) {
+        throw new Error('Multi-AZ cluster is not supported for single-node clusters. got: `ClusterType.SINGLE_NODE`');
```

Comment:
Might want to change the message a bit, according to [the message](https://github.com/go-to-k/aws-cdk/blob/247aa35676b3b1ead91a56fd00e819c67a9a7285/packages/@aws-cdk/aws-redshift-alpha/lib/cluster.ts#L669).

```suggestion
        throw new Error('Multi-AZ cluster is not supported for cluster type single-node');
```

Reply:
Or the following:

```suggestion
        throw new Error('Multi-AZ cluster is not supported for `clusterType` single-node');
```

Or

```suggestion
        throw new Error('Multi-AZ cluster is not supported for single-node `clusterType`');
```

If you have something you think is good...